programmers/budget.js

```diff
@@ -0,0 +1,18 @@
+function solution(d, budget) {
+    d.sort((a,b) => {
+        return a-b
+    });
+    let count = 0;
+    for (let i = 0; i < d.length; i++) {
+        if(d[i] <= budget) {
+            budget-=d[i];
+            count += 1;
+        } else {
+            break;
+        }
+    }
+    return count;
+}
+const d = [1,5,10];
+const budget = 6;
+solution(d,budget);
```

Comment:
저번에 사용하신 forEach를 이용하시면 더 깔끔하게 풀 수가 있습니다.
```suggestion
function solution(d, budget) {
    let result = 0;
    d.sort((a, b) => a - b);
    d.forEach((element) => {
        budget = budget - element;
        if (budget >= 0) {
            result++;
        } 
    });
    return result;
}
```